app/middleware/checkForUser.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable operator-linebreak */
+const User = require('../db/models/user.model');
+const determineCredentials = require('../functions/determineCredentials');
+const pathRequiresUser = require('../functions/pathRequiresUser');
+
+module.exports = async (req, res, next) => {
+  try {
+    // Stuff
+    const path = req.path.split('/')[1];
+    const credentials = determineCredentials(req);
+    const user = await User.findOne({ where: credentials });
+    // Functions
+    const checkIfAnUserExists = () => {
+      if (user) return res.status(403).json({ error: 'A user already exists.' });
+    };
+    const checkIfAnUserDoesNotExist = () => {
+      if (!user) return res.status(404).json({ error: 'User not found.' });
+    };
+    // Conduct check
+    /*
+    If we are hitting an endpoint
```

Comment:
Comment blocks are misaligned. Also make sure to follow the style guide - should look like

```javascript
/**
 * This is a multi
 * line comment.
 */
```

https://github.com/airbnb/javascript#comments--multiline

Reply:
Also make sure these are the max length of 80 characters, I think these are less than that so we can take up fewer lines.

Reply:
Roger!